### PR TITLE
Set Kiali username and password on separate prompt

### LINKED ISSUE #6078: Observability: Kiali passphrase fail to set
Ref: https://istio.io/docs/tasks/observability/kiali/

When you copy and paste following command for setting username and passphrase for Kiali.

```
KIALI_USERNAME=$(read -p 'Kiali Username: ' uval && echo -n $uval | base64)
KIALI_PASSPHRASE=$(read -sp 'Kiali Passphrase: ' pval && echo -n $pval | base64)
```
This will ask for `Kiali Username` and when you enter username, it will exit and never ask you for `Kiali Passphrase`

You may forgot to set passphrase, as above step will not ask you for it. When you proceed further and try to create secret for Kiali username and passphrase.

```
cat <<EOF | kubectl apply -f -
apiVersion: v1
kind: Secret
metadata:
  name: kiali
  namespace: $NAMESPACE
  labels:
    app: kiali
type: Opaque
data:
  username: $KIALI_USERNAME
  passphrase: $KIALI_PASSPHRASE
EOF
```
It throws error:
```
error: error parsing STDIN: error converting YAML to JSON: yaml: line 12: could not find expected ':'
```

### PR DESCRIPTION
**Pull request by shamsher31**
This fix #6078 

Asking for Kiali username and passphrase in separate prompt will set passphrase properly.

[X ] Docs
[ X] User Experience

### CODE REVIEW

File: content/en/docs/tasks/observability/kiali/index.md [hunk 1]
```diff
@@ -40,15 +40,29 @@ authenticate to Kiali.
...
+This will prompt you to provide Kiali Passphrase.
+
+{{< text bash >}}
 $ KIALI_PASSPHRASE=$(read -sp 'Kiali Passphrase: ' pval && echo -n $pval | base64)
 {{< /text >}}
 
 If you are using the Z Shell, `zsh`, use the following to define the credentials:
 
+This will prompt you to provide Kiali Username.
+
```
Comment (ericvn) [inline content/en/docs/tasks/observability/kiali/index.md]: Using _zsh_ on my Mac, the original single text works fine. I'm not sure if you found this failed using _zsh_ or if this is simply following what was done for _bash_. If you aren't seeing the original documented problem in _zsh_, I would revert back to a single copiable box.

Comment (shamsher31) [reply content/en/docs/tasks/observability/kiali/index.md]: I didn't get chance to test it on Mac, but as you suggested I can revert it back for zsh to single copiable box if it's working fine.

File: content/en/docs/tasks/observability/kiali/index.md [hunk 2]
```diff
@@ -40,8 +40,15 @@ authenticate to Kiali.
 
 First, define the credentials you want to use as the Kiali username and passphrase:
```
Comment (frankbu) [inline content/en/docs/tasks/observability/kiali/index.md]:
```suggestion
First, define the credentials you want to use as the Kiali username and passphrase.
```

File: content/en/docs/tasks/observability/kiali/index.md [hunk 3]
```diff
@@ -40,8 +40,15 @@ authenticate to Kiali.
...
 
+This will prompt you to provide Kiali Username.
```
Comment (frankbu) [inline content/en/docs/tasks/observability/kiali/index.md]:
```suggestion
Enter a Kiali username when prompted:
```

File: content/en/docs/tasks/observability/kiali/index.md [hunk 4]
```diff
@@ -40,8 +40,15 @@ authenticate to Kiali.
...
+
 {{< text bash >}}
 $ KIALI_USERNAME=$(read -p 'Kiali Username: ' uval && echo -n $uval | base64)
+{{< /text >}}
+
+This will prompt you to provide Kiali Passphrase.
```
Comment (frankbu) [inline content/en/docs/tasks/observability/kiali/index.md]:
```suggestion
Enter a Kiali passphrase when prompted:
```

**Pull request merged**